RevenueCatUI/Templates/V2/Components/Button/BottomSheetView.swift

```diff
@@ -120,7 +125,6 @@ struct BottomSheetOverlayModifier: ViewModifier {
             )
             .animation(.spring(response: 0.35, dampingFraction: 0.7), value: sheetViewModel)
         }
-        .ignoresSafeArea()
```

Comment:
We already ignore the bottom safe area, no need to do it again:

https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCatUI/Templates/V2/PaywallsV2View.swift#L278